libtransmission/tr-lpd.cc

```diff
@@ -243,109 +261,187 @@ class tr_lpd_impl final : public tr_lpd
 private:
     bool init(struct event_base* event_base)
     {
-        if (initImpl(event_base))
+        ipp_t n_success = NUM_TR_AF_INET_TYPES;
```

Comment:
Since you're not looping through *all* types indiscriminately, but instead only looking at AF_INET and AF_INET6 explicitly, I'd add:
```
static_assert(NUM_TR_AF_INET_TYPES == 2);
```

Reply:
We have this line essentially doing that for us.

https://github.com/transmission/transmission/blob/b49604a238403e8b8aac3b79917ac294f1b47754/libtransmission/tr-lpd.cc#L69